schemas/IDLs/org.gel.models.cva.avro/1.1.0/Transactions.avdl

```diff
@@ -173,7 +182,7 @@ protocol TransactionProtocol {
         /**
         The data to be ingested in CVA compressed
         */
-        union {null, bytes} compressedReportEvents;
+        union {null, bytes} compressedData;
```

Comment:
Should this ever be null?

Reply:
we actually set this to null to return the transaction to the user through the REST API, so we are able to provide an actual valid transaction without of all the data which will be irrelevant.